src/util/patientUtils.js

```diff
@@ -7,8 +7,18 @@ const { findResourceById, findResourcesWithQuery } = require('../database/dbOper
 const { mapResourcesToCollectionBundle, mapArrayToSearchSetBundle } = require('./bundleUtils');
 
 /**
+<<<<<<< HEAD
+<<<<<<< HEAD
  * Wrapper function to get patient data and data
  * requirements for given patient id and map the resources to a collection bundle.
+=======
+ * Wrapper function to get patient data for a given patient id and its data
+ * requirements and map the resources to a collection bundle.
+>>>>>>> 75d7893 (created patientUtils from bundleUtils)
+=======
+ * Wrapper function to get patient data and data
+ * requirements for given patient id and map the resources to a collection bundle.
+>>>>>>> a878b42 (misc changes)
```

Comment:
thoughts?

Reply:
I really want to get the point across that this is a wrapper function - if it's not clear I can add a fourth copy of the doc string

Reply:
:)